gobblin-hive-registration/src/main/java/gobblin/hive/metastore/HiveMetaStoreUtils.java

```diff
@@ -119,6 +119,9 @@ public static Table getTable(HiveTable hiveTable) {
             propValue + " is not a valid Hive table/partition property");
         table.getParameters().put(tokens.get(0), tokens.get(1));
       }
+      if (table.getParameters().containsKey(RUNTIME_PROPS)) {
```

Comment:
Instead of removing it here, can you figure out why the parameter is being added in the first place and remove it there?